### ISSUE TITLE
Craslytics-V8 not reporting error to Firebase Dashboard

### ISSUE THREAD

**Issue by sushant115** [invertase/react-native-firebase#3961]
Hi,
i am using following version, i don't see crashes in Firebase Dashboard.
"@react-native-firebase/app": "8.2.0",
"@react-native-firebase/crashlytics": "8.1.2",

Earlier my project was using fabric after migrating RNFBV8, i removed fabric from project but now crash is not appearing in Firebase Dashboard. and I have Followed the doc for integration, only mismatching is "com.google.gms:google-services", am using 3.0.0 where in doc its mentioned as 4.2.0, if i am upgrading to 4.x.x, i am getting other dependancy error

**Comment by mikehardy** [comment]
We have had reports of projects that started on the original Fabric crashlytics (before Firebase even) that they are not receiving crashes, however we have not been able to reproduce them. I would make absolutely sure you are running in release mode on a real device, that you restart the app afterwards (that's when the crash is sent I believe) and that you wait for "a little while" (5 minutes? 15? it kind of depends on their crash-processing load...) and that you have triggered a real crash - if it's non-fatal you'll need to not filter for just crashes in the dashboard.

You should pursue an update to the current google-services as a separate matter, it may be it has an effect on this problem - I doubt it but wouldn't it feel terrible if that was the root cause?

**Comment by mikehardy** [comment]
Also worth noting that you skipped the template, so I it's not super easy to help. The gradle files in particular are important, as is any firebase.json you have.

**Comment by sushant115** [comment]
Yes, as you mentioned i am running in a real device in release mode also am waiting more than 15 mins, but its not coming..

**Comment by sushant115** [comment]
should i create a new ticket with gradle details?

**Comment by andersonaddo** [comment]
You can do that, or you can upload them here.
Though if you make a new issue and follow the **complete** template there it'd be easier to help

**Comment by sushant115** [comment]
@andersonaddo i am getting this error after upgrading: **java.lang.IllegalArgumentException: Please set your project ID. A valid Firebase project ID is required to communicate with Firebase server APIs: It identifies your project with Google.**

**Comment by harrybui2804** [comment]
I have same problem.
package.json
`"@react-native-firebase/app": "^8.2.0",
 "@react-native-firebase/crashlytics": "^8.1.2",`

firebase.json
`
{
  "react-native": {
    "messaging_ios_auto_register_for_remote_messages": true,
    "messaging_auto_init_enabled": true,
    "messaging_android_notification_channel_id": "tng_office_notification_channel",
    "crashlytics_debug_enabled": true,
    "crashlytics_auto_collection_enabled": true,
    "crashlytics_ndk_enabled": true,
    "crashlytics_disable_auto_disabler": false
  }
}
`
`android/build.gradle`
`
buildscript {
  ....
  repositories {
        google()
        jcenter()
        mavenCentral()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:4.0.0'
        classpath 'com.google.gms:google-services:4.3.3'
        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.0.0'
    }
}
allprojects {
    repositories {
        ...
        google()
       ...
    }
}
`

`android/app/build.gradle`
`
apply plugin: "com.android.application"
apply plugin: 'com.google.gms.google-services'
apply plugin: 'com.google.firebase.crashlytics'
...
`

**Comment by tomasfrancomediamonks** [comment]
I am facing the same issue but only for iOS
